src/kit.ts

```diff
@@ -208,7 +208,7 @@ export async function kitIfCompiler(bin: string, pr?: ProgressReporter): Promise
       const binParentPath = path.dirname(bin);
       const mingwMakePath = path.join(binParentPath, 'mingw32-make.exe');
       if (await fs.exists(mingwMakePath)) {
-        const ENV_PATH = `${binParentPath};${process.env['PATH']}`;
+        const ENV_PATH = `${binParentPath}`;//;${process.env['PATH']}`;
```

Comment:
Can you replace the commented code with a comment that explains why we don't need to store the %PATH% here?